src/prpy/tsr/tsr.py

```diff
@@ -175,8 +199,8 @@ def rpy_within_bounds(rpy, Bw):
         @param Bw bounds on rpy
         @return check a (3,) vector of True if within and False if outside
         """
-        # Unwrap rpy to [-pi, pi].
-        rpy = (numpy.array(rpy) + pi) % (2*pi) - pi
+        # Unwrap rpy to Bw_cont.
+        rpy = wrap_to_interval(rpy, lower=Bw[:, 0])
```

Comment:
At the end of the day, the user only knows about the `Bw` range, which they have specified and may be at some crazy value or loop multiple times or whatever.  So it seems like we should check that we are in this range.

If we can guarantee through properties or something else that  `angle \in Bw_cont iff Bw` is always true, then we can check `Bw_cont` instead.